disable_odoo_online/models/publisher_warranty_contract.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2013 Therp BV (<http://therp.nl>).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import api, models
+
+
+class publisher_warranty_contract(models.AbstractModel):
+    _inherit = 'publisher_warranty.contract'
+
+    @api.multi
+    def update_notification(self, cron_mode=True, context=None):
```

Comment:
Almost seems like Odoo SA is shooting themselves in the foot by using a method available in Community in order to phone home in Enterprise. Solution seems sound though.

Also, think we can drop the `context` arg?